mod/wikirate/lib/javascript/script_collapse.js.coffee

```diff
@@ -88,16 +88,16 @@ findCollapseTarget = (toggle) ->
     else
       $toggle.parent()
 
-  target = $toggle.data('target') || '.collapse'
+  if $toggle.attr("href") != null
+    return $($toggle.attr("href"))
+  target = $toggle.attr("href") || $toggle.data('target') || '.collapse'
   if $toggle.data("collapse")?
     target += $toggle.data("collapse")
-
-  if $toggle.find(target).length
+  else if $toggle.find(target).length
```

Comment:
If you change js or css then you have to update the test database (with `bundle exec  rake wikirate:test:seed:update`) otherwise the tests won't check your changes. The changes above broke the filter form toggle on metric and company pages.
Several points are suspicious there:
1) Line 95 changes `target` which is used in the next if block but you connected that if block with an `else` so that `target` is not used anymore. 
2) This method is supposed to return the element that we want to collapse. "href" seems to be an odd name  choice to contain that information. 
3) you return in line 92 if there is an href attribute but then you check agin in line 93 again for an href attribute